spec/date.html

```diff
@@ -460,21 +460,24 @@ <h1>ToDate ( _date_ )</h1>
     <emu-clause id="sec-temporal-topartialdate" aoid="ToPartialDate">
       <h1>ToPartialDate ( _date_ )</h1>
       <emu-alg>
-        1. Let _result_ be {
-          [[Year]]: *undefined*,
-          [[Month]]: *undefined*,
-          [[Day]]: *undefined*,
-          }.
-        1. If Type(_date_) is Undefined or Null, then
+        1. If Type(_date_) is Object, then
```

Comment:
Minor editorial suggestion, I find early returns easier to read in code, and unless the spec writing style discourages that, I'd probably find it easier here:
```
1. If Type(_date_) is not Object, then
  1. Throw a *TypeError* exception.
1. Let _result_ be...
```